src/app/js/admin/configTenant/ConfigTenantForm.js

```diff
@@ -132,6 +134,27 @@ export const ConfigTenantForm = ({
         setConfigTenant(newConfigTenant);
     };
 
+    const handleThemeChange = (event) => {
+        setIsFormModified(true);
+        setTheme(event.target.value);
+
+        try {
+            const themeValue = themes.find(
+                (value) => value.value === event.target.value,
+            );
+
+            const clonedConfig = _.cloneDeep(JSON.parse(configTenant));
```

Comment:
<!-- Quibbles are very much like nitpick:, except it does not conjure up images of lice and animal hygiene practices.  -->
**quibble (non-blocking):** You should remove `_.cloneDeep` and rename `clonedConfig`...‏